Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:latest
+RUN mkdir -p /src/app
+WORKDIR /src/app
+COPY package.json /src/app/package.json
+RUN npm install
+COPY . /src/app
+EXPOSE 3000
+CMD [ "npm", "start" ]
```

Comment:
Do we need to write a start script fir this to run?

Reply:
indeed... added it

Reply:
Unsure if I missed anything else but I think it looks good